### DISCUSSION TITLE
DDAN中参数p

### DISCUSSION THREAD

**Discussion by sidjif** [jindongwang/transferlearning#427] [Q&A]
您好，王研究员。在拜读您的文章“Transfer Learning with Dynamic Distribution Adaptation”时，您在最后进行参数敏感性分析，提到DDAN中的一个必要参数p，但我在您所写的相关的算法DeepMEDA中没有看到参数p出现在何处。
你在文章DDAN部分写道，模型的损失如下图。参数p体现在最后的拉普拉斯正则化项中。
![image](https://github.com/jindongwang/transferlearning/assets/119156133/24249259-59cd-407d-bb3c-a55b37fdd585)
但在算法中，总的损失计算如下图。
![image](https://github.com/jindongwang/transferlearning/assets/119156133/5389017f-dd28-478b-ba93-01be8bd9eefc)
在算法计算损伤时，并没有体现出拉普拉斯正则化项，不知道是不是体现在weight中了。如果体现在weight中，那么参数p又体现在哪里呢？
可能是我对于算法还没有研究太深，问题可能比较浅显，希望您不吝赐教。

**Comment by wlh-coconut** [comment]
你好，这是贺雯丽的自动回复，您的信息已经收到。祝身体健康，万事如意！                                                      贺雯丽

**Reply by sidjif** [reply]
您好，如果您看到了我的疑问，非常希望您能在闲暇之余解答我的疑问，这对我现在的研究非常重要，我将不胜感激。
